src/Ng/Helpers/EmptySecretInjector.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using NuGet.Services.KeyVault;
+
+namespace Ng.Helpers
+{
+    public class EmptySecretInjector : ISecretInjector
```

Comment:
Copyright header

Also, can you leave an XML doc explaining that the secret injection is done at a higher level and therefore this type doesn't need to do anything

Reply:
Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/3e5c1faea9e244e35d5f69b516630ff94ee90935